Cython/Compiler/Nodes.py

```diff
@@ -3918,7 +3922,7 @@ def generate_keyword_unpacking_code(self, min_positional_args, max_positional_ar
                     num_required_posonly_args += 1
 
         code.putln('Py_ssize_t kw_args;')
-        code.putln('const Py_ssize_t pos_args = PyTuple_GET_SIZE(%s);' % Naming.args_cname)
+        code.putln('const Py_ssize_t pos_args = %s;' % Naming.nargs_cname)
```

Comment:
I think we can get rid of this variable then, too.

Reply:
Sure, it didn't occur to me that it was the same thing.